docker-compose.staging.yml

```diff
@@ -27,6 +27,30 @@ services:
     depends_on:
       - postgres
 
+  hapi:
```

Comment:
@danazkari  plase review the docker config for this new service. We will also need to include hapi in the nginx config and DNS records.